userspace/falco/falco.cpp

```diff
@@ -295,16 +295,32 @@ uint64_t do_inspect(falco_engine *engine,
 			throw sinsp_exception(inspector->getlasterr().c_str());
 		}
 
-		if (duration_start == 0)
-		{
-			duration_start = ev->get_ts();
-		} else if(duration_to_tot_ns > 0)
-		{
-			if(ev->get_ts() - duration_start >= duration_to_tot_ns)
+                #if defined(__x86_64__)
```

Comment:
This needs some attention, too.

Can we find a way to make it work on both the platforms without using `#if#elsif defined(...)` ?